lib/nerves_hub_web/controllers/script_controller.ex

```diff
@@ -0,0 +1,105 @@
+defmodule NervesHubWeb.ScriptController do
+  use NervesHubWeb, :controller
+
+  alias NervesHub.Scripts
+  alias NervesHub.Repo
+
+  plug(:validate_role, [org: :manage] when action in [:new, :create, :delete])
```

Comment:
```suggestion
  plug(:validate_role, [org: :manage] when action in [:new, :create, :edit, :delete])
```
Do we need the edit role in here to make changes?

Reply:
Good catch, also `:update`!